docs/configure-runv-with-containerd-docker.md

```diff
@@ -0,0 +1,57 @@
+# Configure containerd and docker to run with runv
+
+Because ![runv](../README.md) is an [OCI](https://www.opencontainers.org/) compatible runtime, it can work smoothly with other OCI-compatible management tools, namingly [containerd](https://containerd.io/) and [docker](https://www.docker.com/community-edition), by replacing the default runtime [runc](https://github.com/opencontainers/runc).
+
+## Work with containerd
+First, install containerd and save its default configuration to `/etc/containerd/config.toml`, which is the default configuration file containerd looks for.
+
+```
+$containerd config default > /etc/containerd/config.toml
+```
+Then add following lines to `/etc/containerd/config.toml`:
+```
+[plugins.linux]
+  shim = "containerd-shim"
+  no_shim = false
+  runtime = "runv"
+  shim_debug = true
+```
+Now everything is configured and you can start containerd with `containerd` command:
+```
+$containerd
+```
+In another window, test with containerd's client commandline tool `ctr`:
+```
+$ctr pull docker.io/library/busybox:latest
+$ctr run --rm -t docker.io/library/busybox:latest foobar sh
+/ # ls
+bin   dev   etc   home  lib   proc  root  sys   tmp   usr   var
+/ # exit
+```
+
+## Work with docker
+While containerd is a bit low level to users and lacks useful features like network management, docker is more mature and has a lot of fancy features. We choose [docker CE, the community-edition of docker](https://www.docker.com/community-edition) to demonstrate how to configure runv with docker.
+
+First, install docker CE as illustrated in the [docker CE download page](https://www.docker.com/community-edition#/download). As of writing, docker CE version `17.06.1-ce` is used.
+
+Stop docker service (with e.g., `systemctl stop docker`) and then create docker's default configuration file `/etc/docker/daemon.json` with following contents, or add following contents to it if it exits already.
+```
+{
+  "default-runtime": "runv",
+  "runtimes": {
+    "runv": {
+      "path": "runv"
```

Comment:
Please add some comment about hyperstart.
Add some information about where the kernel/initrd should be saved.